include/seqan3/alignment/pairwise/execution/alignment_executor_two_way.hpp

```diff
@@ -172,10 +172,10 @@ class alignment_executor_two_way
                                exec_policy_t const & SEQAN3_DOXYGEN_ONLY(exec)) :
         alignment_executor_two_way{std::move(resrc), std::move(fn)}
     {
-        static_assert(!std::same_as<exec_policy_t, parallel_unsequenced_policy>,
-                      "Parallel unsequenced execution not supported!");
-        static_assert(!std::same_as<exec_policy_t, unsequenced_policy>,
-                      "Unsequenced execution not supported!");
+//        static_assert(!std::same_as<exec_policy_t, parallel_unsequenced_policy>,
+//                      "Parallel unsequenced execution not supported!");
+//        static_assert(!std::same_as<exec_policy_t, unsequenced_policy>,
+//                      "Unsequenced execution not supported!");
```

Comment:
Why comment it out and not just delete it?

Reply:
Because the asserts are correct, the parallel and unsequenced execution not supported yet. But the tests will run in this asserts and will fail.
What would you say? Delete it or let the tests fail, as they should?
Maybe add this to the next level task, where the execution is implemented?